src/Stream/src/BinderRabbitMQ/Config/RabbitConsumerOptions.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using RabbitConfig = Steeltoe.Messaging.RabbitMQ.Config;
+using RabbitCore = Steeltoe.Messaging.RabbitMQ.Core;
+
+namespace Steeltoe.Stream.Binder.Rabbit.Config
+{
+    public class RabbitConsumerOptions : RabbitCommonOptions
+    {
+        public RabbitConsumerOptions()
+            : base()
+        {
+        }
+
+        public bool? Transacted { get; set; }
+
+        public RabbitCore.AcknowledgeMode? AcknowledgeMode { get; set; }
+
+        public int? MaxConcurrency { get; set; }
+
+        public int? Prefetch { get; set; }
+
+        public int? BatchSize { get; set; }
+
+        public bool? DurableSubscription { get; set; }
+
+        public bool? RepublishToDlq { get; set; }
+
+        public RabbitCore.MessageDeliveryMode? RepublishDeliveryMode { get; set; }
+
+        public bool? RequeueRejected { get; set; }
+
+        public List<string> HeaderPatterns { get; set; }
+
+        public int? RecoveryInterval { get; set; }
+
+        public bool? Exclusive { get; set; }
+
+        public bool? MissingQueuesFatal { get; set; }
+
+        public int? QueueDeclarationRetries { get; set; }
+
+        public long? FailedDeclarationRetryInterval { get; set; }
+
+        public string ConsumerTagPrefix { get; set; }
+
+        public int? FrameMaxHeadroom { get; set; }
+
+        public RabbitConfig.ContainerType? ContainerType { get; set; } = RabbitConfig.ContainerType.DIRECT;
+
+        public string AnonymousGroupPrefix { get; set; }
+
+        internal void PostProcess(RabbitConsumerOptions defaultOptions = null)
+        {
+            if (!Transacted.HasValue)
+            {
+                Transacted = defaultOptions != null ? defaultOptions.Transacted : false;
```

Comment:
```suggestion
                Transacted = defaultOptions?.Transacted ?? false
```
Functionally equivalent, easier to read for me, optional change for all of these